src/client/providers/settings/settings.ts

```diff
@@ -0,0 +1,8 @@
+import { UserSettings } from "../../types.js";
+
+export const settings: UserSettings = {
+    userId: 1,
+    settings: {
+        toolApprovalRequired: true,
```

Comment:
let's set this to default false until we figure out how to store the settings in a file

this way we can merge the PR soon and keep the scope of the PR small